test/unit/test_linux_local_file_cache.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2012-2020 Snowflake Computing Inc. All right reserved.
+#
+
+import os
+
+import snowflake.connector.auth as auth
+from snowflake.connector.compat import IS_LINUX
+
+HOST_0 = "host_0"
+HOST_1 = "host_1"
+USER_0 = "user_0"
+USER_1 = "user_1"
+CRED_0 = "cred_0"
+CRED_1 = "cred_1"
+
+CRED_TYPE_0 = "ID_TOKEN"
+CRED_TYPE_1 = "MFA_TOKEN"
+
+
+def get_credential(sys, user):
+    return auth.TEMPORARY_CREDENTIAL.get(sys.upper(), {}).get(user.upper())
+
+
+def test_basic_store():
+    if not IS_LINUX:
+        return
```

Comment:
Please use a skip here instead, we can attach a reason to that, so that'd be a little more verbose than this.

Reply:
Will do!